tablo/models.py

```diff
@@ -432,25 +451,32 @@ def _validate_where_clause(self, where):
         parsed = self._parse_where_clause(where)
 
         if parsed is None:
-            return True
+            return
         elif parsed[1]:
-            return False  # More than one statement means SQL injection.
-        else:
-            return self._validate_fields(parsed[0])
+            raise SQLInjectionError('Invalid where clause')
```

Comment:
I don't know about calling it an SQLInjectionError. Calling it that implies intent, which may or may not be there. It could just be a bad or mistyped `where` clause. InvalidFieldsError works just as well here.